website/app/initializers/showdown-extensions.js

```diff
@@ -93,13 +93,8 @@ export function initialize(/* application */) {
           language = 'shell';
         }
 
-        if (Prism.languages[language]) {
-          // if the specified language is available in Prism we highlight the codeblock
-          highlightedCodeBlock = Prism.highlight(codeblock, Prism.languages[language], language) + end;
```

Comment:
More thinking out loud here, but the previous function here used the specified language to highlight the block so I was wondering if the new approach might not highlight different languages correctly. In practice it seems to be working ok though, at least looking at `hbs` and `js` code blocks.

Reply:
Yeah lang still gets added as css selector and the Prism.highlightAllUnder() call in the show controller looks for language for syntax highlighting.  I will test a couple additional langs and make sure this new method is exactly the same.

Reply:
Nice. That's what I think I understood from reading the docs, but was good to confirm.

Reply:
Double verified and we can add additional language support in ./website/ember-cli-build.js I tested by adding support for html and scss and highlighting was applied with old prism calls and our new one triggered in didRender.  Highlighting seems to be theme dependent though.  We are passing theme: 'dracula' to our 'ember-prism' prop in appConfig and this particular theme doesn't support every language. Eg. tested with php and json and no highlighting support available before this change or after. So in the future if we want to support more langs we will need to make the the prism theme we are using has appropriate css selectors.